components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java

```diff
@@ -3035,4 +3035,17 @@ public static void resetAuthenticationContext(AuthenticationContext context) {
         context.setRetrying(false);
         context.setCurrentAuthenticator(null);
     }
+
+    /**
+     * Check whether the JIT provisioning enhanced feature is enabled.
+     *
+     * @return true if the JIT provisioning enhanced features is enabled else return false.
+     */
+    public static boolean isEnhancedFeature() {
```

Comment:
Better to say what enhanced feature we're enabling in the method name.

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3597/commits/6e1589981f43782e7824b22d065010c4ef34b01d